models/sms_types.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+const (
+	// TypeSms is used to consume the short message service and send messages to given mobile numbers
+	TypeSms = "sms"
+	// TypeCheck TODO
+	TypeCheck = "check"
+	// TypeCheckSms TODO
+	TypeCheckSms = "checksms"
+	// TypeHlr TODO
+	TypeHlr = "hlr"
```

Comment:
Type definition seems to be different on Japanese docs and English docs. 
It is ok about this since we use only sms currently. 
I'll ask media4u on it too and share it the answer to team. 

**English**
![image](https://user-images.githubusercontent.com/70126599/99027946-d94be180-25b1-11eb-8e39-9eb518e58aae.png)

**Japanese**
![image](https://user-images.githubusercontent.com/70126599/99027975-ef59a200-25b1-11eb-9383-24af221b946c.png)

Reply:
Media4u confirmed that values on Japanese docs is correct. I think available values are these for type parameter.
I uploaded updated documentations. 
https://drive.google.com/drive/u/0/folders/1Wc9NmMECra4e0mULX3GZ0Cs4I6VTPPrI 
```
sms
check
checksm
shlr
hlrsms
```